packages/jupyter-chat/src/chat-commands/types.ts

```diff
@@ -67,7 +68,6 @@ export interface IChatCommandProvider {
    */
   handleChatCommand(
     command: ChatCommand,
-    currentWord: string,
-    replaceCurrentWord: (newWord: string) => void
+    inputModel: IInputModel
   ): Promise<void>;
```

Comment:
Can you also pass the input model to `getChatCommands()` method above on line 60? The `currentWord: string` argument should be replaced with `inputModel: IInputModel`.

Reply:
Done